### PR TITLE
Added Webstorm App

### PR DESCRIPTION
**Pull request by stovv**
## Description
The ability to open Webstorm Workspace (several projects) with option for closing all other projects

## Type of change

- [x] New script command

## Screenshot
webstorm raycast open workspace script

## Dependencies / Requirements
- WebStorm

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/apps/webstorm/open-workspace.template.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# @raycast.argument1 { "type": "text", "placeholder": "Workspace", "optional": true }
+# @raycast.argument2 { "type": "text", "placeholder": "Close others? [yes/no]", "optional": true }
+#
+# Documentation:
+# @raycast.description Open webstorm projects
+# @raycast.author Daniel Stovv
+# @raycast.authorURL https://github.com/stovv
+#
+# Configuration
+# 1. Please make sure for webstorm installed and webstorm script available in PATH
```
Comment (unnamedd) [inline commands/apps/webstorm/open-workspace.template.sh:25]:
```suggestion
# 1. Please make sure for WebStorm installed and WebStorm script available in $PATH
```

**Comment by unnamedd** [comment]
@stovv, any intention to work on this PR again?

**Comment by dehesa** [comment]
Let's wait till the end of the week and if by Monday @stovv hasn't said anything I will close this down.

**Comment by stovv** [comment]
@unnamedd @dehesa Thank you for your comments! I'll be done by the end of the week

**Comment by stovv** [comment]
@unnamedd Please check new changes

**Review by unnamedd — APPROVED** [review]
Hello @stovv,

thanks a lot for your changes! 
It looks good to me! Thanks a lot!

**Comment by dehesa** [comment]
Looks great @stovv . Thank you for the contribution!

**Pull request merged**